packages/webapp/src/components/Form/Switch/index.tsx

```diff
@@ -0,0 +1,64 @@
+/*
+ *  Copyright 2022-2024 LiteFarm.org
+ *  This file is part of LiteFarm.
+ *
+ *  LiteFarm is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  LiteFarm is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ *  GNU General Public License for more details, see <https://www.gnu.org/licenses/>.
+ */
+
+import { ChangeEvent } from 'react';
+import clsx from 'clsx';
+import { useTranslation } from 'react-i18next';
+import styles from './styles.module.scss';
+import { Main } from '../../Typography';
+
+interface SwitchProps {
+  checked?: boolean;
+  onChange?: (event: ChangeEvent<HTMLInputElement>) => void;
+  label?: string;
+  leftLabel?: string;
+  isToggleVariant?: boolean;
+  disabled?: boolean;
+}
+
+const Switch = ({
+  checked,
+  onChange,
+  label,
+  leftLabel,
+  isToggleVariant,
+  disabled,
+  ...props
+}: SwitchProps) => {
+  const { t } = useTranslation();
+
+  return (
+    <div className={styles.container} {...props}>
+      {leftLabel && <Main>{leftLabel}</Main>}
+      <label className={styles.switch}>
+        <input onChange={onChange} checked={checked} type="checkbox" disabled={disabled} />
+        <span
+          className={clsx(
+            styles.track,
+            styles.round,
```

Comment:
It's working great!

Do you know why this span has two classNames not used anywhere else?
I think it will still work even if we remove `round` (and make adjustments)...🤔

Reply:
Did they want... to keep this option as an eventual possibility? 😂 Actually it's a bit cool! But I'll remove.

<img width="152" alt="Screenshot 2024-06-14 at 10 40 52 AM" src="https://github.com/LiteFarmOrg/LiteFarm/assets/82857964/d58a701b-84d7-4583-b961-c458470a2b43">

Reply:
Ah, that makes sense!!!
I like the square one too 😂 
Thank you Joyce!